.env.example

```diff
@@ -5,8 +5,8 @@ PORT=
 
 # DB env
 DB_URL='url_to_db'
-DB_USER='ange'
-DB_PASSWORD='mecoco'
+DB_USER='postgres'
+DB_PASSWORD='divine@2020'
```

Comment:
seems like you pushed your secret info ??

Reply:
No,  that's just the sample of .env